### PR TITLE
[Yarn script]Move host dir under yarn monitor, simplify yarn script exit log

### PR DESCRIPTION
**Pull request by mzmssg**
1. Move all host folders under yarn monitor

2. Simplify yarn exit log, origin log is  too repetitive.
image

### CODE REVIEW

File: src/rest-server/src/templates/yarnContainerScript.mustache [hunk 1]
```diff
@@ -26,29 +26,15 @@ export PAI_DEFAULT_FS_URI={{{ hdfsUri }}}
...
+  printf "[DEBUG] EXIT signal received in yarn container, performing clean up action...\n"
 
-  debug_log "$handler" "write exit code to file"
-  debug_log "$handler" "yarn container exit code: $rc"
-  debug_log "$handler" "exit code file path: /var/lib/hadoopdata/nm-local-dir/nmPrivate/$APP_ID/$CONTAINER_ID/$CONTAINER_ID.pid.exitcode"
+  printf "[DEBUG] Write exit code $rc to file /var/lib/hadoopdata/nm-local-dir/nmPrivate/$APP_ID/$CONTAINER_ID/$CONTAINER_ID.pid.exitcode.\n"
   echo $rc > "/var/lib/hadoopdata/nm-local-dir/nmPrivate/$APP_ID/$CONTAINER_ID/$CONTAINER_ID.pid.exitcode"
 
-  debug_log "$handler" "clean the container code"
-  rm -fr tmp/pai-root/code 2>/dev/null
```
Comment (hao1939) [inline src/rest-server/src/templates/yarnContainerScript.mustache]: We are not going to delete the code? Will Yarn clean it up?

File: src/rest-server/src/templates/yarnContainerScript.mustache [hunk 2]
```diff
@@ -275,9 +264,9 @@ docker run --name $docker_name \
   $nvidia_devices \
   --device=/dev/fuse \
   --security-opt apparmor:unconfined \
-  --volume /tmp/pai-root/alive/$APP_ID:/alive \
-  --volume /tmp/pai-root/alive/$APP_ID/yarn_$CONTAINER_ID:/alive/yarn_$CONTAINER_ID:ro \
-  --volume /tmp/pai-root/log/$APP_ID/$CONTAINER_ID:/pai/log \
+  --volume $container_local_dir/$alive_dir:/alive \
+  --volume $container_local_dir/$alive_dir/yarn_$CONTAINER_ID:/alive/yarn_$CONTAINER_ID:ro \
+  --volume $container_local_dir/$log_dir:/pai/log \
```
Comment (hao1939) [inline src/rest-server/src/templates/yarnContainerScript.mustache]: How could we debug now?
Will the logs be archived on HDFS?

File: src/rest-server/src/templates/yarnContainerScript.mustache (hunk 1 above)
Comment (mzmssg) [reply src/rest-server/src/templates/yarnContainerScript.mustache]: Yes, all folders under yarn local directory will be cleaned.

File: src/rest-server/src/templates/yarnContainerScript.mustache [hunk 3]
```diff
@@ -57,11 +43,13 @@ PS4="+[\t] "
 trap exit_handler EXIT
 
 
-mkdir -p "/tmp/pai-root/alive/$APP_ID"
+alive_dir="tmp/pai-root/alive"
```
Comment (fanyangCS) [inline src/rest-server/src/templates/yarnContainerScript.mustache]: >alive_dir [](start = 0, length = 9)

should we link this folder to a yarn folder so Yarn can clean it up?

Comment (mzmssg) [reply src/rest-server/src/templates/yarnContainerScript.mustache]: YARN wil not delete symlink folders.

**Pull request merged**